bot/resources/tags/questions.md

```diff
@@ -0,0 +1,3 @@
+**Asking Good Questions**
```

Comment:
This needs to change to comply with the news tags slash command
#2419 is your friend